CHANGELOG.md

```diff
@@ -1,5 +1,65 @@
 # Changelog
 
+## [1.0.0-rc.13] - 2025-03-04
+
+The key change is the removal of `github.com/nspcc-dev/neofs-api-go/v2` module.
+When updating, app code may break in places importing components of this module.
+However, it is expected that this will only affect the system code: regular apps
+following the lib recommendations will most likely not require edits.
+
+Several code elements have been marked as deprecated. It is strongly
+recommended to update all places. All deprecated parts will be removed in the
+upcoming major release.
+
+Go 1.23 is the minimum required now. Watch for dep changes in `go.mod` file.
```

Comment:
Usually this is mentioned in the log below as well with references.

Reply:
good place here to me, enough info is provided

Reply:
Links are relevant anyway and we have them.

Reply:
what links